package.json

```diff
@@ -31,11 +31,11 @@
     "assets-webpack-plugin": "7.0.*",
     "babel-loader": "^9.1.3",
     "chalk": "^4.0.0",
-    "clean-webpack-plugin": "^4.0.0",
     "css-loader": "^6.7.0",
     "css-minimizer-webpack-plugin": "^5.0.0",
     "fastest-levenshtein": "^1.0.16",
     "mini-css-extract-plugin": "^2.6.0",
+    "multimatch": "^5",
```

Comment:
The latest multimatch version is 7.0. but as of 6.0, this package is shipped as ESM which makes it unusable in webpack-encore (see #1312)